### ISSUE TITLE
0.8.0rc9: gitlab-ci-linux-build-120 - Bionic-Mate rock64 build fails to boot.

### ISSUE THREAD

**Issue by Xeddius** [ayufan-rock64/linux-build#353]
Rock64 4GB,
8G SD card, 16G SD, 64G Class 10 SD card.
Fails to boot with error:
```
mali-utgard ff300000.gpu: failed to get pvtm
devfreq ff300000.gpu: Couldn't update frequency transition information
```
Then hangs, flashes a kernel panic on screen, and reboots.
Pretty sure it's the same issue as https://github.com/ayufan-rock64/linux-build/issues/347
Haven't been able to get the build to work at all thus far.

**Comment by b0b33140** [comment]
It's the same for me on rock64

**Comment by b0b33140** [comment]
The lxde version works but I think there is a problem with the graphical acceleration.

**Comment by ayufan** [comment]
It is disabled for now. To at least make it booting :)

**Comment by b0b33140** [comment]
bionic-mate-rock64-0.8.0rc10-1125-armhf.img.xz
does not work, I have a black screen at the start with the mouse cursor. :(

**Comment by rhens1** [comment]
An observation, bionic-mate-rock64-0.8.0rc(7, 8 &9) failed to sync on my rock64v2 4GB (failed at 720P@75Hz).  However all bionic-mate-rock64-0.8.0 builds, but rc10, DID boot and connection through a SSH tunnel and VNC was easy.  Rc10 threw page fault errors and would NOT boot, but panicked and died after a couple of reboots.  This was using EMMC or class10 SanDisk µSDs (also powered SDs).  The µSDs were in the USB3 port.  Bionic-mate-rockpro64-0.8.0rc(7, 8 &9) builds are flawless on my rock64v2 (4GB).  OK, Firefox (Not Chromium) with YouTube has issues along with the screen power level (for Pinebook??), but the HDMI A/V is great and fully functional.  A LAMP (3 domain + userdirs) server in the background does not significantly slow this little unit down.  Has anyone else reported trying the rockpro64 images on a rock64v2?  Ayufan, this has me perplexed.  Daily upgrades have not been a problem, but an upgrade to another release candidate - this is going to be an issue at some point.  Feel free to delete my comment.  I thought it might help

**Comment by ayufan** [comment]
This is no longer relevant. The latest 0.8.2 fixes that.

**Comment by DmitryMyadzelets** [comment]
This issue is not solved in the `buster-minimal 0.9.4`, as I mention in [the other issue](https://github.com/ayufan-rock64/linux-build/issues/347).
